apps_product_creator/views/view_product_product.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright (C) 2016-Today: Odoo Community Association (OCA)-->
+<!--License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).-->
+<odoo>
+
+    <record id="product_product_form_view" model="ir.ui.view">
+        <field name="name">product.product.form (in github_product_creator)</field>
+        <field name="model">product.product</field>
+        <field name="priority" eval="100"/>
+        <field name="inherit_id" ref="product.product_normal_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook" position="inside">
+                <page string="Module Informations" id="module_informations" attrs="{'invisible':[('odoo_module_version_id','=',False)]}">
```

Comment:
Replace `id=` with `name=`.

Also I'm noticing that the tab "Module Informations" is missing for `product.template`: it is only present for `product.product`. Is it made on purpose?